OpenRA.Mods.Common/SpriteLoaders/ShpTSLoader.cs

```diff
@@ -118,7 +118,9 @@ bool IsShpTS(Stream s)
 			while (w == 0 && h == 0 && f++ < imageCount);
 
 			s.Position = start;
-			return type < 4;
+
+			// Type 18 is a special case for RA2 low bridges.
```

Comment:
The downside of course is that it raises the chances of falsely identifying non-shp images with unfortunate coincidences in byte patterns as shp.